portal/templates/portal_wrapper.html

```diff
@@ -89,6 +89,9 @@
                             {% if user and user.has_role(ROLE.STAFF.value, ROLE.STAFF_ADMIN.value) %}
                             <li class="resources-menu-item"><a href="{{PORTAL}}/resources">{{ _("Resources") }}</a></li>
                             {% endif %}
+                            {% if user and user.has_role(ROLE.CLINICIAN.value, ROLE.STAFF.value, ROLE.STAFF_ADMIN.value) %}
+                                <li class="eproms-substudy"><a href="{{PORTAL}}/substudy-tailored-content">{{ _("EMPROS Resources") }}</a></li>
```

Comment:
Should this be `EMPRO` not `EMPRO`**S**?
```suggestion
                                <li class="eproms-substudy"><a href="{{PORTAL}}/substudy-tailored-content">{{ _("EMPRO Resources") }}</a></li>
```

Reply:
thank you for catching that, Ivan! Fixed